Resources/Prototypes/GameRules/roundstart.yml

```
@@ -182,6 +188,12 @@
       roleLoadout:
       - RoleSurvivalNukie
       components:
+      # Sunrise edit start
+      - type: GhostPanelAntagonistMarker
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 7

(indentation)